src/components/Home/ProjectCard/index.tsx

```diff
@@ -6,7 +6,36 @@ import { useNavigate } from 'react-router-dom';
 
 const ProjectCard = (props: any) => {
```

Comment:
Can you create a interface of props passing to this component? Avoid passing props but instead pass every value separately